src/Components/ArticlePage.jsx

```diff
@@ -30,9 +35,9 @@ function ArticlePage() {
         })
     }
     
-    
-    
-
+    if (articleExist) {
```

Comment:
Naming of the state used to store the error message is confusing. Wouldn't it make more sense to name this state something along the lines of error/setError? As it doesn't specifically refer to the article, and syntactically the truthy error condition reading as `if (articleExist)` makes it seems like a successful request has been made, which is the opposite to what's actually happening in our code if this criteria is met 🤔